transcom/mymove
# B 20752 int

## [B-20752](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20752)

## Summary

This PR is a follow-up to [B-20753](https://github.com/transcom/mymove/pull/13486), which simply adds the new NTS handle in/out fields to the move details page.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the application as a customer, and create a new move with an NTS and NTSR shipments.
2. Access the office application as a TOO, and pull up the shipment you created.
3. While on the move details page, ensure the Requested delivery date is present for the NTS Shipment. (See screenshot 1)
4. While on the move details page, ensure the Requested pickup date is present for the NTS-release Shipment. (See screenshot 1)
5. Edit the shipments and fill out the Requested delivery date and Requested pickup date for the NTS and NTS-release, respectively.
6. When back on the move details page, ensure the new values are populated in move details. (See screenshot 2)

### Frontend


## Screenshots
<img width="947" alt="image" src="https://github.com/user-attachments/assets/544f09bf-1df5-4c37-bbec-cbfcb591ca63">
<img width="947" alt="image" src="https://github.com/user-attachments/assets/781a5938-56f9-4759-99cd-3a90a4941d2b">
